docs/css/README.md

```diff
@@ -1,14 +1,201 @@
+
+To supplement `styled-components`, this boilerplate also uses
+[`sanitize.css`](https://github.com/jonathantneal/sanitize.css)
+to make browsers render all elements more consistently and in line with modern standards,
+it's a modern alternative to CSS resets. More info available on the [`sanitize.css` page](sanitize.md).
+
+## CSS Support
+
+We support and recommend the use of [`styled-components`](#components).
+We also support the use of CSS [stylesheets](#stylesheet).
+
+There are many ways to style web applications, unfortunately, we cannot support them all.
+However, you can integrate the following styling methods:
+- [CSS Modules](#css-modules)
+- [Sass](#sass)
+
+## `styled-components`
+
+Below creates two styled react components (`<Title>`, `<Wrapper>`) and renders them
+as children of the `<Header>` component:
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+
+// Create a <Title> react component that renders an <h1> which is
+// centered, palevioletred and sized at 1.5em
+const Title = styled.h1`
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+`;
+
+// Create a <Wrapper> react component that renders a <section> with
+// some padding and a papayawhip background
+const Wrapper = styled.section`
+  padding: 4em;
+  background: papayawhip;
+`;
+
+// Use them like any other React component – except they're styled!
+function Header(props) {
+  return (
+    <Wrapper>
+      <Title>Hello {props.name}, this is your first styled component!</Title>
+      ...
+    </Wrapper>
+  );
+}
+
+```
+
+*(The CSS rules are automatically vendor prefixed, so you don't have to think about it!)*
+
+For further examples see the
+[official documentation](https://github.com/styled-components/styled-components)
+
+## Stylesheet
+
+[Webpack](https://webpack.js.org/) allows you to import more than JavaScript.
+Using the [`css-loader`](https://webpack.js.org/loaders/css-loader/) you can import CSS
+into a JavaScript:
+
+**`Button.css`**
+```css
+.danger {
+  background-color: red;
+}
+```
+
+**`Button.js`**
+```js
+import React from 'react';
+import './Button.css'; // Tell Webpack that Button.js uses these styles
+
+class Button extends Component {
+  render() {
+    // You can use them as regular CSS styles
+    return <button className="danger">Click me</button>;
+  }
+}
+```
+
+## [CSS Modules](https://github.com/css-modules/css-modules)
+
+### Setup
+
+Modify [`webpack.base.babel.js`](../../internals/webpack/webpack.base.babel.js#L30)
+to look like the following:
+
+```diff
+{
+  test: /\.css$/,
+  exclude: /node_modules/,
+- use: ['style-loader', 'css-loader'],
++ use: [
++   'style-loader',
++   {
++     loader: 'css-loader',
++     options: {
++       modules: true,
++     }
++   },
++ ]
+}
+```
+
+### Usage
+
+The syntax is very similar to using a [Stylesheet](#stylesheet)
+and this often catches people out.
+The key difference in CSS Modules is that you import to a variable.
```

Comment:
Form my perspective "import to a variable" is fine, otherwise "import _styles_ to a variable". 😄